frontend/src/plaquettes/Template.js

```diff
@@ -1,230 +1,87 @@
-    // this.workspace.addChild(this.container);
+    this.initializeTemplate();
   }
 
-  // Render the template control buttons
-  renderTemplateControlButtons() {
-    this.isDragging = true;
-    this.container.name = 'template';
-    // Create the buttons
-    const { x, y } = this.workspace.mainButtonPosition;
-    this.clearButton = new Button('Clear', x, y + 50);
-    this.clearButton.on('click', () => {
-      // Clear the template
-      this.clearButton.visible = false;
-      this.templateButton.visible = true;
-      this.makeTileButton.visible = false;
-      this.isDragging = false;
-      this.templateQubits.forEach((qubit) => {
-        qubit.changeColor('black');
-      });
-      // Clear the template qubits
-      this.templateQubits = [];
-      // Remove listeners
-      this.app.view.removeEventListener(
-        'mousedown',
-        this.mouseDownCreateTemplateArea
-      );
-      this.app.view.removeEventListener(
-        'mousemove',
-        this.mouseDragResizeTemplateArea
-      );
-      this.app.view.removeEventListener(
-        'mouseup',
-        this.mouseUpFinishTemplateArea
-      );
-      this.rectangle.visible = false;
-      notification(this.app, 'Step 1: Drag to define a template area');
-    });
-
-    this.makeTileButton = new Button(
-      'Step 2: Confirm Template',
-      100,
-      170,
-      'darkgreen'
-    );
-    this.makeTileButton.on('click', () => {
-      if (this.templateQubits.length === 0) {
-        notification(this.app, 'Template requires +3 qubits');
-        return;
-      }
-      this.makeTileButton.visible = false;
-      this.clearButton.visible = false;
-      this.isDragging = false;
-      // Remove listeners
-      this.app.view.removeEventListener(
-        'mousedown',
-        this.mouseDownCreateTemplateArea
-      );
-      this.app.view.removeEventListener(
-        'mousemove',
-        this.mouseDragResizeTemplateArea
-      );
-      this.app.view.removeEventListener(
-        'mouseup',
-        this.mouseUpFinishTemplateArea
-      );
-      notification(this.app, 'Step 3: Click on 3+ qubits to make a plaquette');
+  /**
+   * Initialize the template
+   * @function initializeTemplate
+   * @returns {void}
+   */
+  initializeTemplate = () => {
+    // Create the plaquettes and template
+    this.selectQubitsButton.on('click', () => {
       this.app.view.addEventListener('click', this.selectQubit);
-      this.plaquetteButton.visible = true;
-      this.plaquetteButton.on('click', () => {
-        // Create the plaquettes and tile
-        this.createPlaquette();
-        this.workspace.addChild(this.container);
-        // Clear the selected qubits
-        this.selectedQubits = [];
-        notification(this.app, 'Step 4: Define the circuit');
+      this.unselectQubitsButton.visible = true;
+      this.selectQubitsButton.visible = false;
+      this.createPlaquetteButton.visible = false;
+    });
+    this.unselectQubitsButton.on('click', () => {
```

Comment:
Nice. I'm planning on adding undo/redo buttons in general (for the whole app), but this is good to have until that's implemented.